contracts/VToken.sol

```diff
@@ -296,7 +307,17 @@ contract VToken is Ownable2StepUpgradeable, VTokenInterface, ExponentialNoError,
      * @return rate The supply interest rate per block, scaled by 1e18
      */
     function supplyRatePerBlock() external view override returns (uint256) {
-        return interestRateModel.getSupplyRate(_getCashPrior(), totalBorrows, totalReserves, reserveFactorMantissa);
+        if (totalBorrows == 0) {
```

Comment:
This is the supply rate for stable rates, not for variable rates. But in `borrowRatePerBlock` we are still returning the rate for the variable interest model. It seems incoherent for me.

We might provide two new methods returning the borrow and the supply rates per block for the stable rate model, and keep the current ones returning the variable rates. I suppose the UI will need to offer this information in a separate way.

Reply:
1. This is the supply rate of the protocol(per block) which is calculated by considering stable as well as variable borrows. 
2. `borrowRatePerBlock` returns the variable borrow rate per block, so I added a method to return stable borrow rate per block.
3. Also, updated the PoolLens, to return stable rate per block also in vToken metadata.

Can verify the changes at [commit](https://github.com/VenusProtocol/isolated-pools/pull/132/commits/a8c2a085c97f3b5b813c35757af27c5baf56edfa).